package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "todo-list-app",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] Your project is looking good from the code base but I was unable to test the functionalities because I found no script that needs to command webpack to start your app. 
Please try adding ``` "serve": "webpack serve"``` in the package.json scripts and run ```npm run serve``` on your terminal to ensure webpack can execute your app successfully.